### PR TITLE
Set end_of_picture flag for AV1 encoded input.

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

File: talk/owt/sdk/base/customizedvideoencoderproxy.cc [hunk 1]
```diff
@@ -347,6 +347,11 @@ int32_t CustomizedVideoEncoderProxy::Encode(
     encoded_frame._frameType = encoder_buffer_handle->meta_data_.is_keyframe
                                   ? webrtc::VideoFrameType::kVideoFrameKey
                                   : webrtc::VideoFrameType::kVideoFrameDelta;
+    if (encoder_buffer_handle->meta_data_.last_fragment) {
```
Comment (taste1981) [inline talk/owt/sdk/base/customizedvideoencoderproxy.cc:350]: just: 

info.end_of_picture = encoder_buffer_handle->meta_data_.last_fragment;

**Review by dvrogozh — COMMENTED** [review]
Av1 encoding works for me after this fix. Can you explain what the concept of "last_fragment" is about?

**Comment by taste1981** [comment]
> Av1 encoding works for me after this fix. Can you explain what the concept of "last_fragment" is about?

|last_fragment| signals whether current bitstream is the last slice/tile/segment of an entire codec picture(frame). When slice/tile based encoding is not enabled, |last_fragment| should always be configured to "true".

File: talk/owt/sdk/base/customizedvideoencoderproxy.cc (hunk 1 above)
Comment (jianjunz) [reply talk/owt/sdk/base/customizedvideoencoderproxy.cc]: Done.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**